server/src/main/resources/com/broadcom/lsp/cobol/service/delegates/completions/LanguageKeywords.txt

```diff
@@ -91,6 +91,7 @@ BELOW=
 BIF=
 BINARY=
 BINARY=Specifies a binary data item.<br>[Read more](https://www.ibm.com/support/knowledgecenter/SS6SG3_6.2.0/com.ibm.cobol62.ent.doc/PGandLR/ref/rlddecom.html)<br>\u00A9 Copyright IBM Corporation 1994, 2019.<br>U.S. Government Users Restricted Rights - Use, duplication or disclosure restricted by GSA ADP Schedule Contract with IBM Corp.
+BIND=
```

Comment:
Same duplication check (and remove) in LanguageKeywords.txt

Reply:
Also, I believe you need to support COLORING for IDMS keywords,  you need to ADD  those newly-added keywords into "COBOL.tmLanguage".josn as well

Reply:
The initial agreement with the IDMS team was to provide the grammar, there was nothing about the coloring. Implementing it right now requires setting an additional meeting to clarify the responsibilities. I don't want to block the PR because of bureaucracy, so I would recommend extending the coloring as a separate activity. 
Please, approve the changes.

Reply:
@temanbrcom ok coloring can be done separate I like to approve, but there are **still duplications**.